herd/tests/instructions/AArch64.kvm/A005.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A005
+{
+[PTE(x)]=(oa:PA(x),valid:0);
+0:X1=x;
+}
+ P0          | P0.F           ;
+ LDR W0,[X1] | MOV X2,#1      ;
+L0:          | ADR X3,L0      ;
+ MOV X3,#1   | MSR ELR_EL1,X3 ;
+             | ERET           ;
+ forall(0:X2=1 /\ 0:X3=1)
```

Comment:
For litmus' not to issue a warning it would be nice not to use register `X3` both as address and integer and to use the `W` form of registers when they hold integers:
```suggestion
 LDR W0,[X1] | MOV W2,#1      ;
L0:          | ADR X3,L0      ;
 MOV W4,#1   | MSR ELR_EL1,X3 ;
             | ERET           ;
 forall(0:X2=1 /\ 0:X4=1)
```